components/AppRegistration/AppRegistrationLogin.tsx

```diff
@@ -0,0 +1,14 @@
+import { oauthUrl } from '../../appRegistrationEffects';
+export default function AppRegistrationLogin() {
+    return (
+        <div className="app-registration-login">
+            <div className="login-image">
+                <img src="/img/login.svg" alt="Login" width="128px" height="128px" />
+            </div>
```

Comment:
```suggestion
            <div className="login-image" />
```

Reply:
possibly can also be className={styles.loginImage} if you change to .module.scss